src/components/views/elements/PersistedElement.js

```diff
@@ -167,9 +167,9 @@ export default class PersistedElement extends React.Component {
             width: parentRect.width + 'px',
             height: parentRect.height + 'px',
         });
-    }
+    }, 100, {trailing: true, leading: true});
```

Comment:
Curious, why did this need to be added?

Reply:
This is just so the PersistedElement (floating iframe) doesn't update too frequently, it was causing the whole resize operation to lag